kde-devel@thinktatil:~/src/kde-workspace/ksmserver/themes$ git diff .
diff --git a/ksmserver/themes/default/main.qml b/ksmserver/themes/default/main.qml
index 4dea4b9..8924fac 100644
--- a/ksmserver/themes/default/main.qml
+++ b/ksmserver/themes/default/main.qml
@@ -186,14 +186,17 @@ PlasmaCore.FrameSvgItem {
 
     Text {
         id: automaticallyDoLabel
-        text: " "
         font.pointSize: theme.desktopFont.pointSize >= 10 ? theme.desktopFont.pointSize*0.9 : theme.desktopFont.pointSize
         color: theme.textColor
+        wrapMode: Text.WordWrap
+        horizontalAlignment: Text.AlignRight
         anchors {
             top: parent.top
             topMargin: realMarginTop
             right: parent.right
             rightMargin: realMarginRight
+            left: parent.left
+            leftMargin: realMarginLeft
         }
     }
 
